File: app/src/main/java/com/OverSadBoy/samurairise/model/database/ItemDao.java
package com.OverSadBoy.samurairise.model.database;

import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;
import androidx.room.Query;

import java.util.ArrayList;
import java.util.List;

@Dao
public interface ItemDao {
    @Insert
    void addAlarm(Item item);

    @Query("DELETE FROM Item WHERE id = :id")
    void deleteAlarm(int id);

    @Query("SELECT * FROM" + " Item")
    List<Item> getAlarms();
}

File: app/src/main/java/com/OverSadBoy/samurairise/model/database/ItemDao.kt
package com.OverSadBoy.samurairise.model.database

import androidx.room.Dao
import androidx.room.Delete
import androidx.room.Insert
import androidx.room.Query

@Dao
interface ItemDao {
    @Insert
    fun addAlarms(item: Item)

    @Delete
    fun deleteAlarm(id: Int)

    @Query("SELECT * FROM " + "alarms")
    fun getAllAlarms(): List<Item>
}